app/src/main/java/pl/gittobefit/network/ConnectionToServer.java

```diff
@@ -18,6 +18,7 @@
     public final WorkoutFormsServices WorkoutFormsServices;
     public final TrainingServices trainingServices;
     private static volatile ConnectionToServer INSTANCE;
+    public final String PREFIX_VIDEO_URL = "https://static.fabrykasily.pl/atlas/";
```

Comment:
Póki co jest to okej, ale po code review i wrzuceniu filmików będzie można to usunąć.